samples/SocialSample/Startup.cs

```diff
@@ -329,20 +330,20 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
                 var user = context.User;
 
                 // This is what [Authorize] calls
-                // var user = await context.Authentication.AuthenticateAsync(AuthenticationManager.AutomaticScheme);
+                // var user = await context.AuthenticateAsync(AuthenticationManager.AutomaticScheme);
 
                 // This is what [Authorize(ActiveAuthenticationSchemes = MicrosoftAccountDefaults.AuthenticationScheme)] calls
```

Comment:
That was a name that slipped through, was supposed to be renamed to AuthenticationScheme to match the policy :(